src/pages/settings/Settings.tsx

```diff
@@ -9,34 +9,95 @@ import {
   useNotify,
   Spinner,
   CustomLayout,
+  useToastNotification,
 } from "@canonical/react-components";
-import SettingForm from "./SettingForm";
 import NotificationRow from "components/NotificationRow";
 import HelpLink from "components/HelpLink";
 import { queryKeys } from "util/queryKeys";
-import { fetchConfigOptions } from "api/server";
+import { fetchConfigOptions, updateSettings } from "api/server";
 import { useQuery } from "@tanstack/react-query";
-import type { ConfigField } from "types/config";
-import ConfigFieldDescription from "pages/settings/ConfigFieldDescription";
 import { toConfigFields } from "util/config";
 import PageHeader from "components/PageHeader";
 import { useSupportedFeatures } from "context/useSupportedFeatures";
 import { useServerEntitlements } from "util/entitlements/server";
-import type { ClusterSpecificValues } from "components/ClusterSpecificSelect";
 import { useClusteredSettings } from "context/useSettings";
-import type { LXDSettingOnClusterMember } from "types/server";
 import { useProjects } from "context/useProjects";
-import { getDefaultProject } from "util/loginProject";
+import {
+  type UserSetting,
+  getConfigFieldClusteredValue,
+  getUserSettings,
+} from "util/settings";
+import {
+  getSettingRow,
+  getUserSettingInputRow,
+  getAddSettingButton,
+} from "pages/settings/SettingsRow";
+import ResourceLabel from "components/ResourceLabel";
+import type { MainTableRow } from "@canonical/react-components/dist/components/MainTable/MainTable";
 
 const Settings: FC = () => {
   const [query, setQuery] = useState("");
+
+  const [userSettings, setUserSettings] = useState<UserSetting[]>([]);
+
   const notify = useNotify();
+  const toastNotify = useToastNotification();
+
   const {
     hasMetadataConfiguration,
     settings,
     isSettingsLoading,
     settingsError,
   } = useSupportedFeatures();
+
+  const { data: projects = [] } = useProjects();
+
+  useEffect(() => {
+    if (userSettings.length === 0 && !isSettingsLoading) {
+      setUserSettings(getUserSettings(settings?.config ?? {}, projects));
+    }
+  }, [settings, projects]);
```

Comment:
```suggestion
  }, [settings, projects, isSettingsLoading]);
```
This seems to be missing from dependencies